Sources/LiveKit/Publications/RemoteTrackPublication.swift

```diff
@@ -184,7 +204,15 @@ private extension RemoteTrackPublication {
 
     var userCanModifyTrackSettings: Bool {
         // adaptiveStream must be disabled and must be subscribed
-        !isAdaptiveStreamEnabled && subscribed
+        if kind == .video && isAdaptiveStreamEnabled {
+            log("Adaptive stream is enabled, cannot change video track settings", .warning)
+            return false
+        }
+        if !(preferSubscribed || subscribed) {
```

Comment:
I believe this is no longer true. they should be able to change settings anytime